packages/application-components/src/components/modal-pages/form-modal-page/form-modal-page.story.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text, boolean } from '@storybook/addon-knobs/react';
+import withReadme from 'storybook-readme/with-readme';
+import { Formik } from 'formik';
+import { PORTALS_CONTAINER_ID } from '@commercetools-frontend/constants';
+import {
+  Spacings,
+  TextInput,
+  TextField,
+  SecondaryButton,
+} from '@commercetools-frontend/ui-kit';
+import { number } from '@storybook/addon-knobs/dist/deprecated';
```

Comment:
Is this import correct? 🤔

Reply:
Whoops, VScode automatically did this import and I didn't notice 😄
Good catch!